projects/gateway2/krtcollections/uniqueclients.go

```diff
@@ -64,15 +64,20 @@ func labeledRole(role string, labels map[string]string) string {
 	return fmt.Sprintf("%s%s%d", role, xds.KeyDelimiter, utils.HashLabels(labels))
 }
 
+// note: if `ns“ is empty, we assume the user doesn't want to use pod locality info, so we won't modify the role.
```

Comment:
I thought it is the env var that controls pod locality usage. This comment is a bit confusing in relation to the new env var.

Reply:
It does but not in the part of the code. 
This part is not aware of the env var